File: fixtures/sample-repo/src/php/TokenService.php
<?php
namespace App\Auth;

use App\Shared\Clock;

class TokenService
{
    public function issueToken(string $user): string
    {
        return $this->helperSalt() . ':' . $user;
    }

    private function helperSalt(): string
    {
        return 's3cret';
    }
}

File: fixtures/sample-repo/src/csharp/TokenService.cs
using System.Collections.Generic;

namespace Sample.Auth
{
    public class TokenService
    {
        public string IssueToken(string user)
        {
            return HelperSalt() + ":" + user;
        }

        private string HelperSalt()
        {
            return "s3cret";
        }
    }
}
